### ISSUE TITLE
`os-error-alias` does not include `os.error`

### ISSUE THREAD

**Issue by Avasam** [astral-sh/ruff#7580]
`os.error` is an alias for `OSError`, but is not standardized as part of https://docs.astral.sh/ruff/rules/os-error-alias/

```py
>>> import os; OSError is os.error
True
```

```py
import os

try:
    pass
except (WindowsError, os.error):
    pass
```

becomes:
```py
import os

try:
    pass
except (OSError, os.error):
    pass
```

expected:
```py
import os  # This import is now unused and can be removed

try:
    pass
except OSError:
    pass
```

~~This would be a deviation from pyupgrade, I have opened the same request in parallel: https://github.com/asottile/pyupgrade/issues/892~~
Edit: This was just added in https://github.com/asottile/pyupgrade/pull/893
